partner_contact_type_default/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# SPDX-FileCopyrightText: 2023 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+{
+    "name": "Partner Contact Type Default",
+    "summary": """
+        Set 'contact' as default type when creating a partner as a contact of another
+        partner.""",
+    "version": "16.0.1.0.0",
+    "category": "Uncategorized",
```

Comment:
most modules in [OCA/partner-contact](https://github.com/OCA/partner-contact) seems to use the "Customer Relationship Management" category.